### ISSUE TITLE
Add parser control code

### ISSUE THREAD

**Issue by iceflow19** [iceflow19/jazzy#5]
Add the parser control code to the master rule, and labeling rule, allowing for navigation forward and backwards in the parse stream.  Also change the op calling in each.

For an example refer to whitespacer by HostileFork.  This issue needs to be implemented after #1 and #4

**Comment by iceflow19** [comment]
Completed.  May need revising, but we can open another issue for that.
